### PR TITLE
[13.0][MIG] barcode_action

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module barcode_action to v13.

cc ~ @ForgeFlow @etobella @JoanSForgeFlow

### CODE REVIEW

**Review by JoanSForgeFlow — APPROVED** [review]
Tested on local, LGTM :+1:

File: barcode_action/wizard/barcode_action.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class BarcodeAction(models.TransientModel):
+    _name = "barcode.action"
+    _inherit = "barcodes.barcode_events_mixin"
```
Comment (LoisRForgeFlow) [inline barcode_action/wizard/barcode_action.py:6]: quick one: `2020-03-09 15:01:47,324 158 WARNING openerp_test odoo.models: The model barcode.action has no _description`

Comment (ps-tubtim) [inline barcode_action/wizard/barcode_action.py:6]: Please add `_description` for fix ci/runbot.

Comment (AdriaGForgeFlow) [reply barcode_action/wizard/barcode_action.py:6]: :+1:

**Review by ps-tubtim — APPROVED** [review]

**Review by etobella — APPROVED** [review]
LGTM

**Pull request merged**